spring-data-vmware-gemfire/src/main/java/org/springframework/data/gemfire/listener/annotation/ContinuousQuery.java

```diff
@@ -5,12 +5,10 @@
 
 package org.springframework.data.gemfire.listener.annotation;
 
-import java.lang.annotation.Documented;
-import java.lang.annotation.ElementType;
-import java.lang.annotation.Inherited;
-import java.lang.annotation.Retention;
-import java.lang.annotation.RetentionPolicy;
-import java.lang.annotation.Target;
+import org.apache.geode.cache.query.ExcludedEvent;
+
+import java.lang.annotation.*;
```

Comment:
Something went wrong with the formatting here.... We should never do import * on any package

Reply:
Fixed